<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParticipantTableWidget</class>
 <widget class="QWidget" name="ParticipantTableWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>555</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="participantsFilterLineEdt">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>פילטר</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addParticipantBtn">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="participantstable.qrc">
         <normaloff>:/assets/add.png</normaloff>:/assets/add.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="assignParticipantBtn">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="participantstable.qrc">
         <normaloff>:/assets/assign.png</normaloff>:/assets/assign.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeParticipantBtn">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="participantstable.qrc">
         <normaloff>:/assets/remove.png</normaloff>:/assets/remove.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="participantsTableWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>454</height>
      </size>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustIgnored</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="participantstable.qrc"/>
 </resources>
 <connections/>
</ui>
